src/main/java/bio/terra/service/filedata/flight/ingest/FileIngestFlight.java

```diff
@@ -54,7 +54,8 @@ public FileIngestFlight(FlightMap inputParameters, Object applicationContext) {
         RetryRuleRandomBackoff lockDatasetRetry =
             new RetryRuleRandomBackoff(500, appConfig.getMaxStairwayThreads(), 5);
 
-        RetryRuleRandomBackoff fileSystemRetry = new RetryRuleRandomBackoff(500, appConfig.getMaxStairwayThreads(), 5);
+        RetryRuleRandomBackoff fileSystemRetry =
+            new RetryRuleRandomBackoff(500, appConfig.getMaxStairwayThreads(), 5);
```

Comment:
There's no harm in having separate objects, but the same retry rule can be used on multiple steps. 

I wonder if we should have a data repo version of this class that builds this random back off retry. If we ever decide those are the wrong values we will have to change it in about 100 places :)